Extension/Global/Strings.cs

```diff
@@ -24,6 +24,12 @@ public static class Strings
         public const string SETTINGS_PREFER_SHIELD_TOOLTIP = "Prefer infantry with shields when upgrading to infantry troop classes.";
         public const string SETTINGS_RANGED_PREFERENCE_NAME = "Prefer ranged with specific weapons";
         public const string SETTINGS_RANGED_PREFERENCE_TOOLTIP = "Prefer specific ranged weapons when upgrading to ranged troop classes. (0 for no preference, 1 for Bows, 2 for crossbows)";
+        public const string SETTINGS_BUFFER_DAILY_COST_NAME = "Buffer days to pay daily costs";
+        public const string SETTINGS_BUFFER_DAILY_COST_TOOLTIP = "Number of days to pay daily costs upgrading should not use.";
```

Comment:
This has to be reworded

Reply:
Keep enough gold for X days of troop wages? (thanks Autotrader)